nats-server-config-reloader.yaml

```diff
@@ -0,0 +1,117 @@
+package:
+  name: nats-server-config-reloader
+  version: "0.17.1"
+  epoch: 0
+  description: "NATS server configuration reloader utility"
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nack.git
+      tag: v${{package.version}}
+      expected-commit: 03e5e930036a5baacc12ef604cd57ad6ff6ada96
+
+  - uses: go/build
+    with:
+      packages: ./cmd/nats-server-config-reloader
+      output: nats-server-config-reloader
+      ldflags: -X main.BuildTime=$(date -u +%Y-%m-%dT%H:%M:%S%z) -X main.GitInfo=$(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short HEAD)$(git diff --quiet || echo "-dirty") -X main.Version=${{package.version}}
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/local/bin
+      cp -r ${{targets.destdir}}/usr/bin/nats-server-config-reloader ${{targets.destdir}}/usr/local/bin/nats-server-config-reloader
+      chmod 755 ${{targets.destdir}}/usr/local/bin/nats-server-config-reloader
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compat package for ${{package.name}}"
+    pipeline:
+      - runs: |
+          ln -sf /usr/local/bin/nats-server-config-reloader ${{targets.contextdir}}/nats-server-config-reloader
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - runs: |
+            /usr/bin/nats-server-config-reloader --help
+            /usr/local/bin/nats-server-config-reloader --help
+
+  - name: ${{package.name}}-oci-entrypoint
+    description: Entrypoint for ${{package.name}}
+    pipeline:
+      - runs: |
+          cp cicd/assets/entrypoint.sh ${{targets.contextdir}}/entrypoint.sh
+          chmod +x ${{targets.contextdir}}/entrypoint.sh
+      - uses: strip
+    test:
+      pipeline:
+        - name: "Check expected scripts exist"
+          runs: |
+            stat /entrypoint.sh
+
+update:
+  enabled: true
+  github:
+    identifier: nats-io/nack
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - bash
+        - nats-server
+        - curl
+        - jq
+  pipeline:
+    - name: "Verify basic installation"
+      runs: |
+        nats-server-config-reloader --version
```

Comment:
Can we `| grep ${{package.version}}` here to see if we properly set the version with LDFLAGS?